Ryujinx/Ui/GLRenderer.cs

```diff
@@ -0,0 +1,422 @@
+﻿using Gdk;
+using OpenTK;
+using OpenTK.Graphics;
+using OpenTK.Graphics.OpenGL;
+using OpenTK.Input;
+using Ryujinx.Configuration;
+using Ryujinx.Graphics.OpenGL;
+using Ryujinx.HLE;
+using Ryujinx.HLE.Input;
+using Ryujinx.Ui;
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading;
+
+namespace Ryujinx.Ui
+{
+    public class GLRenderer : GLWidget
+    {
+        private const int TouchScreenWidth = 1280;
+        private const int TouchScreenHeight = 720;
+        private const int TargetFps = 60;
+
+        public ManualResetEvent WaitEvent { get; set; }
+
+        public bool IsActive   { get; set; }
+        public bool IsStopped  { get; set; }
+        public bool IsFocussed { get; set; } = false;
```

Comment:
```suggestion
        public bool IsFocused { get; set; }
```
A bool is always false by default.